tests/Mail/MailMailerTest.php

```diff
@@ -209,8 +210,13 @@ public function testGlobalToIsRespectedOnAllMessages()
         });
 
         $this->assertSame('taylor@laravel.com', $sentMessage->getEnvelope()->getRecipients()[0]->getAddress());
-        $this->assertStringNotContainsString('dries@laravel.com', $sentMessage->toString());
-        $this->assertStringNotContainsString('james@laravel.com', $sentMessage->toString());
```

Comment:
This assertion was irrelevant as BCC addresses would not be in the string

Reply:
That was the whole point to prove with this assertion.

Reply:
But current tests seem all good to me 👍